jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -15,156 +14,170 @@ pipeline {
         ansiColor('xterm')
     }
 
-
     parameters {
         string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch or GitHash for CORTX Cluster scripts', trim: true)
-        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re/', description: 'Repository for CORTX Cluster scripts', trim: true)
+        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repository for CORTX Cluster scripts', trim: true)
         string(name: 'CORTX_TAG', defaultValue: 'main', description: 'Branch or GitHash for generaing CORTX container images', trim: true)
         string(name: 'OS_VERSION', defaultValue: 'CentOS 7.9.2009 x86_64', description: 'Operating system version', trim: true)
         string(name: 'REGION', defaultValue: 'ap-south-1', description: 'AWS region', trim: true)
-        string(name: 'KEY_NAME', defaultValue: 'automation-key', description: 'Key name', trim: true)
-        string(name: 'EBS_VOLUME_COUNT', defaultValue: '9', description: 'EBS volumes to attach onto nodes. Currently maximum 25 disks are supported', trim: true)
-        string(name: 'EBS_VOLUME_SIZE', defaultValue: '10', description: 'EBS volume size in GB to attach onto nodes', trim: true)
+        string(name: 'KEY_NAME', defaultValue: 'devops-key', description: 'Key name', trim: true)
         string(name: 'COMMUNITY_USE', defaultValue: 'yes', description: 'Only use during community deployment', trim: true)
-        
-        // Please configure ROOT_PASSWORD, ACCESS_KEY and SECRET_KEY parameters in Jenkins job configuration.
-
-    stages {
+        string(name: 'VOLUME_COUNT', defaultValue: '9', description: 'EBS volume', trim: true)
+        string(name: 'VOLUME_SIZE', defaultValue: '10', description: 'EBS volume size', trim: true)
+        string(name: 'INSTANCE_COUNT', defaultValue: '4', description: 'Instance count', trim: true)
```

Comment:
we can change description to `EC2 instance count`

Reply:
fixed